packages/react-components/react-card/src/stories/CardFocusMode.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+
+import { Title3, Text } from '@fluentui/react-text';
+
+import { makeStyles, shorthands } from '@griffel/react';
+import { SampleCard } from './SampleCard.stories';
+
+const useStyles = makeStyles({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    ...shorthands.gap('30px'),
+  },
+});
+
+export const Focus = () => {
```

Comment:
can we name this better to express the purpose ? On the first sight I've no idea what `Focus` does in terms of those stories

Reply:
Would a 1:1 match of the prop name be best? I felt having a more human name would be better than the actual prop name as this is what the user sees on the storybook sidebar.
![image](https://user-images.githubusercontent.com/39736248/166662016-a003b5c7-cde1-45a3-a69f-a03261c82310.png)

Reply:
I'm bit out of the loop on latest style guides regarding story file names naming and exposing story names in public docsite for storybook.

regarding the image you added - AFAIR this wont be rendered in react-components suite right? if not I think " I felt having a more human name would be better than the actual prop name as this is what the user sees on the storybook sidebar" can be dismissed. Either way I'd probably name it `FocusMode` to have alignment with file name and the general purpose of that component ? 

wdyt ?

Reply:
Good point, I'll make it 1:1